packages/z2s/src/compiler.ts

```diff
@@ -596,20 +596,29 @@ export class Compiler {
         this.#nameMapper.columnName(table, column)
       ];
     const serverType = serverColumnSchema.type;
-    if (
-      !serverColumnSchema.isEnum &&
-      (serverType === 'date' ||
+    if (!serverColumnSchema.isEnum) {
+      if (
+        serverType === 'date' ||
         serverType === 'timestamp' ||
+        serverType === 'timestamp without time zone'
```

Comment:
I believe these two branches are the same again, and can be merged.

Reply:
ah yep. done